ui/src/containers/SampleViewContainer.js

```diff
@@ -68,6 +69,17 @@ class SampleViewContainer extends Component {
       }
     });
 
+    const phaseControl = (
```

Comment:
Ill merge this one for now, but there might be some discussion among the people here on where to display this. We might need to make it configurable :)